batch/create-batch-tables.sql

```diff
@@ -21,6 +21,7 @@ CREATE TABLE IF NOT EXISTS `jobs` (
   PRIMARY KEY (`id`)
 ) ENGINE = InnoDB;
 CREATE INDEX jobs_user ON jobs (user);
+CREATE INDEX jobs_batch ON jobs (batch_id);
 
 CREATE TABLE IF NOT EXISTS `jobs-parents` (
```

Comment:
jobs-parents has primary key job_id, parent_id.  That lets you efficiently query job_id, or (job_id, parent_id), but not parent_id by itself.  We should have job_id be the primary key (which it is: each entry should be uniquely identified by the job id), and a separate index on parent_id.

Reply:
done